src/App.js

```diff
@@ -4,13 +4,14 @@ import Announcements from "./components/Announcements";
 import Inventory from "./components/Inventory";
 import Orders from "./components/Orders";
 import NavigationBar from "./components/NavigationBar";
-import { Route, Switch } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
       <NavigationBar />
       <Switch>
+        {/*<Route exact path="/" component={Announcements} />*/}
```

Comment:
Let's uncomment this so that we can have it default to announcements

Reply:
don't uncomment, just set path of <Route component={Announcements} path="/announcements" /> to 
Route component={Announcements} path="/" 
 and move it to after
Route component={Inventory} path="/inventory" 
Route component={Orders} path="/orders"